templates/web/base/report/new/form_user_loggedin.html

```diff
@@ -54,7 +54,7 @@
     [% INCLUDE 'report/_show_name_label.html' %]
 
 [% IF NOT c.user.phone_verified AND NOT c.cobrand.call_hook('disable_phone_number_entry') %]
-    <label for="form_phone">[% loc('Phone number (optional)') %]</label>
+    <label for="form_phone">[% loc('Phone number') %] <span id="phone_label_optional_tag">[% loc('(optional)') %]</span></label>
```

Comment:
This is the form for logged in users; you'll need equivalent in `report/form/user_loggedout_by_email.html` for logged out people. And note there it is more complicated because of possibility of signing up to updates by SMS, so it already has 'js-optional-phone' in use as the ID there, which you might want to use as the ID here as well.